src/msg.fbs

```diff
@@ -194,18 +194,26 @@ table KeyValue {
   value: string;
 }
 
-table Fetch {
-  id: uint;
+// Note this represents The WHOLE header of an http message, not just the key
```

Comment:
What's the advantage of doing this vs separating the status/method out of this table?

Reply:
The idea is to reuse this message for server-side messages too. If url/method/status is put into the table, there will be duplicate code to extract those in the server/client.